base/database/dbtest.go

```diff
@@ -0,0 +1,18 @@
+package database
+
+import "github.com/jinzhu/gorm"
+
+type TestTable struct {
+	Name  string
+	Email string
+	gorm.Model
```

Comment:
We don't use `gorm.Model` and `AutoMigrate`, so we should use the same style for testing and using.

Reply:
Removed `gorm.Model`, but keeping the `AutoMigrate` as a way to simply generate table in the test database, without requiring  inclusion of the test table into the production database or custom SQL.